exporter/metrics.py

```diff
@@ -1,8 +1,8 @@
 from prometheus_client import Gauge
 
-cpu_utilization_gauge = Gauge("cpu_utilization", "CPU utilization percentage")
-cpu_frequency_gauge = Gauge("cpu_frequency", "CPU frequency in MHz")
-cpu_temperature_gauge = Gauge("cpu_temperature", "CPU temperature in Celsius")
+cpu_utilization_gauge = Gauge("stagent_cpu_utilization_overall_percentage", "System-wide CPU utilization as a percentage")
+cpu_frequency_gauge = Gauge("stagent_cpu_frequency_current_mhz", "CPU frequency in MHz")
+cpu_temperature_gauge = Gauge("stagent_cpu_coretemp_sensor_1_celsius", "Core temperature from Sensor 1 in Celsius")
```

Comment:
@mccooeyc11
I have also refactored the naming and descriptions of Gauges to make them more comprehensive for the frontend to read them.